### DISCUSSION TITLE
Attached JS plugin to live component

### DISCUSSION THREAD

**Discussion by tpohare** [filamentphp/filament#13698] [Help]
### Package

Form builder

### Package Version

v3.2.96

### How can we help you?

I am trying to attach a grammar checker (webspellchecker) to a live Rich Editor. 

The component is straightforward

```
        return RichEditor::make("Summary")
            ->required()
            ->live()
            ->hiddenLabel($hide_label)
            ->hint(function ($state) {
                $words = $state ? explode(' ', $state) : [];

                return 'Word count: '.count($words);
            });
```

For normal inputs, the spellcheck runs fine but I am running into an issue with live inputs where as soon as something is entered an update triggers so the spellchecker is lost and the input loses focus, Is there an event I can listen for to avoid losing focus / re-apply the spellchecker?

Thanks for your help

**Comment by leandrocfe** [comment]
Maybe you could use onBlur or debounce
https://filamentphp.com/docs/3.x/forms/advanced#reactive-fields-on-blur

**Comment by tpohare** [comment]
Thanks for the suggestion but neither seem to help

Trying onBlur: The editor works as expected until it loses focus and then the component reloads without the spellchecker, it also means the word count doesn't update until the editor loses focus

Trying debounce 500: The editor works as expected until the debounce time is reached and then the component reloads without the spellchecker, it also means the word count doesn't update until the editor loses focus
